lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -93,6 +93,7 @@ class TwilioConnectionTransport extends StateMachine {
       reconnectBackOffJitter: RECONNECT_BACKOFF_JITTER,
       reconnectBackOffMs: RECONNECT_BACKOFF_MS,
       sdpFormat: getSdpFormat(options.sdpSemantics),
+      trackSwitchOff: true, // mpatwardhan_fix_before_merge: should this be false by default?
```

Comment:
@makarandp0 This flag is not necessary. I think the `track_switch_off` MSP should always be created. Can you confirm with Jose?

Reply:
@manjeshbhargav - I have posted a question [regarding this](https://twilio.slack.com/archives/C35JTJHJP/p1562025963119100?thread_ts=1561480456.095900&cid=C35JTJHJP) but I am still wondering shouldn't this be a client option, just like DominantSpeaker? Basically if client is not going to use these track_switch_off notifications they could save bandwidth by setting something like: `trackSwitchOffNotifications: false` as part of connect options.